index.html

```diff
@@ -2137,15 +2200,15 @@ <h2>Integrations</h2>
       <h3>Permissions Policy</h3>
 
       <p>
-        This specification defines a feature that controls whether {{TCPSocket}} and {{UDPSocket}}
-        classes may be created.
+        This specification defines a feature that controls whether {{TCPSocket}}, {{UDPSocket}}
+        and {{TCPServerSocket}} classes may be created.
 
       <p>
         The feature name for this feature is "<dfn
           data-dfn-for="policy-controlled feature"><code>direct-sockets</code></dfn>"`.
 
       <p>
-        The default allowlist for this feature is `'self'`.
+        The default allowlist for this feature is `'none'`.
```

Comment:
q: is `none` the correct default for IWAs? Or do we define it somehow else?

Reply:
IWAs override the default allowlist for all features to `'none'` so it somewhat doesn't matter.

Reply:
I'd leave this as `none` then.